### ISSUE TITLE
Airship Contact | @ReactMethod `contactEditContactSubscriptionLists` should be named as `contactEditSubscriptionLists`

### ISSUE THREAD

**Issue by Angad-Tiwari** [urbanairship/react-native-airship#491]
# Preliminary Info

### What Airship dependencies are you using?
"@ua/react-native-airship": "^15.2.0"

### What are the versions of any relevant development tools you are using?
```
IDE - Visual Studio Code 
Version: 1.76.2 (Universal)
Commit: ee2b180d582a7f601fa6ecfdad8d9fd269ab1884
Date: 2023-03-14T17:54:09.061Z
Electron: 19.1.11
Chromium: 102.0.5005.196
Node.js: 16.14.2
V8: 10.2.154.26-electron.0
OS: Darwin x64 22.3.0
Sandboxed: No
```

    "react": "^17.0.2",
    "react-native": "^0.68.2"

### Other Info
Android > BuildConfig.java
`public static final boolean IS_NEW_ARCHITECTURE_ENABLED = false;`
or 
`isTurboModuleEnabled = false`

# Report

### What unexpected behavior are you seeing?
In Android App, while subscribing/unsubscribing the preferences from the contact subscription, it throws an uncaught exceptions.
 
![Screenshot_1679484163](https://user-images.githubusercontent.com/87816236/226894229-9c2231eb-5fdd-488f-993a-6627263412be.png)

### What is the expected behavior?
Should able subscribe/unsubscribe from the contact subscription
`Airship.contact.editSubscriptionLists().subscribe(subscriptionId, SUBSCRIPTION_SCOPE_APP).apply()`

### What are the steps to reproduce the unexpected behavior?
On subscribing/unsubscribing the contact subscription issue got occurred using the latest 15.2.0
`Airship.contact.editSubscriptionLists().subscribe(subscriptionId, SUBSCRIPTION_SCOPE_APP).apply()`
`Airship.contact.editSubscriptionLists().unsubscribe(subscriptionId, SUBSCRIPTION_SCOPE_APP).apply()`

### Where do we have the issue?
- [AirshipContact.ts](https://github.com/urbanairship/react-native-airship/blob/main/src/AirshipContact.ts)
<img width="752" alt="Screenshot 2023-03-22 at 4 22 51 PM" src="https://user-images.githubusercontent.com/87816236/226891109-bfaf152b-2921-4dc7-afc0-887ab31c15a9.png">

AirshipContact.ts calling `this.module.contactEditSubscriptionLists` whereas @ReactNative method name defined not matching.
Refer below files with module's method definition @ReactNative - 

1. [AirshipSpec.kt](https://github.com/urbanairship/react-native-airship/blob/main/android/src/oldarch/java/com/urbanairship/reactnative/AirshipSpec.kt)
<img width="468" alt="Screenshot 2023-03-22 at 4 25 00 PM" src="https://user-images.githubusercontent.com/87816236/226891459-e0653191-6c83-4c5a-9cb6-6f791178f038.png">

2. [AirshipModule.kt](https://github.com/urbanairship/react-native-airship/blob/main/android/src/main/java/com/urbanairship/reactnative/AirshipModule.kt)
<img width="763" alt="Screenshot 2023-03-22 at 4 25 38 PM" src="https://user-images.githubusercontent.com/87816236/226891733-0203d80d-dbd1-4751-a14e-e2a937a9f76a.png">

### Do you have logging for the issue?
` ERROR  TypeError: _this3.module.contactEditSubscriptionLists is not a function. (In '_this3.module.contactEditSubscriptionLists(operations)', '_this3.module.contactEditSubscriptionLists' is undefined)`

**Comment by rlepinski** [comment]
The fix is merged into main but the subscription list issue you reported is not yet resolved. I found an issue in the underlying SDK, so I need to get a fix out for that before I will do a release on this plugin.

**Comment by Angad-Tiwari** [comment]
Thanks @rlepinski for this fix. 
Looking forward for the contact subscription list issue fix ASAP, as this is one of our blocker for the app release.

Will close this issue once plugin is been updated.
👍

**Comment by rlepinski** [comment]
Fixed in 15.2.1

**Comment by Angad-Tiwari** [comment]
The issue is resolved in v15.2.1

Thanks @rlepinski
